src/main/java/com/github/awwkoala/selenium/page/DynamicControlsPage.java

```diff
@@ -0,0 +1,65 @@
+package com.github.awwkoala.selenium.page;
+
+import org.assertj.core.api.Assertions;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class DynamicControlsPage extends BasePage {
+
+  @FindBy(id = "btn")
+  private WebElement button;
+
+  @FindBy(id = "checkbox")
+  private WebElement checkbox;
+
+  @FindBy(id = "message")
+  private WebElement message;
+
+  public DynamicControlsPage(WebDriver driver) {
+    super(driver);
+  }
+
+  @Override
+  public void goTo() {
+    driver.get("http://theinternet.przyklady.javastart.pl/dynamic_controls");
+  }
+
+  private boolean isCheckboxVisible() {
+    try {
+      waitUtils.waitUntilIsVisible(checkbox);
+      return checkbox.isDisplayed();
+    } catch (TimeoutException e) {
```

Comment:
If you'd like to return false instead of throwing exception, I think it's generic approach and it could be moved to some Utils Method. WebElementUtils isElementVisible(WebElement) ?